src/log4net.ElasticSearch.Tests/UnitTests/UnitTestContext.cs

```diff
@@ -31,14 +31,29 @@ public static UnitTestContext Create(int bufferSize = BufferSize, bool? failSend
             var errorHandler = new ErrorHandlerStub();
 
             var appender = new TestableAppender(repository)
+            {
+                Lossy = false,
+                BufferSize = bufferSize,
+                ConnectionString = ConnectionString,
+                ErrorHandler = errorHandler,
+                FailSend = failSend,
+                FailClose = failClose,
+                RollingIndexNameDateFormat = "yyyy-MM-dd"
+            };
+
+            appender.AddFieldNameOverride(new FieldNameOverride

```

Comment:
Does this mean we need to release a major version and document this field overrides are required by our users who are updating?

Reply:
No, it should be completely backwards compatible.  Might want to add this to the wiki, but all of these new settings have a default value and thus aren't required.

Reply:
Thanks for clarifying